nucypher/cli/commands/worklock.py

```diff
@@ -219,14 +212,32 @@ def claim(general_config, worklock_options, registry_options, force, hw_wallet):
                                                                 registry=registry,
                                                                 show_balances=True)
 
-    # TODO: Show amount of tokens to claim
+    bidder = worklock_options.create_bidder(registry=registry, hw_wallet=hw_wallet)
 
+    unspent_bid = bidder.available_compensation
+    if unspent_bid:
+        emitter.echo(f"Before claiming your tokens, note that WorkLock didn't use your bid completely.\n"
+                     f"An unspent amount of {prettify_eth_amount(unspent_bid)} is available for refund.")
+        if not force:
+            click.confirm(f"Collect unspent bid amount for bidder {worklock_options.bidder_address}?", abort=True)
```

Comment:
Is this more of a *do you want to proceed* type question? It seems as if 'N' would just not collect the unspent bid and continue, but in actuality, it aborts the entire process.

Would it make sense for something like the following:

```suggestion
            click.confirm(f"Before claiming your NU tokens for {worklock_options.bidder_address}, you will be refunded your unspent bid amount. Would you like to proceed?", abort=True)
```